### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by karakarakaraff**

### CODE REVIEW

**Comment by bobbilee19** [comment]
Hey let me know what specifically to look at in this / review. link me. as u can see there are 110 files changed.

**Comment by karakarakaraff** [comment]
I just wanted you to make sure it's set up correctly. Specifically, are the models, routes and schema good to go before I start serializing? Here are the links:

* app/models/item.rb: https://github.com/karakarakaraff/open-to-do/blob/d8bb594e4a2b2fd184b17d8eb3ca21cd18f5febc/app/models/item.rb
* app/models/list.rb: https://github.com/karakarakaraff/open-to-do/blob/d8bb594e4a2b2fd184b17d8eb3ca21cd18f5febc/app/models/list.rb
* app/models/user.rb: https://github.com/karakarakaraff/open-to-do/blob/d8bb594e4a2b2fd184b17d8eb3ca21cd18f5febc/app/models/user.rb
* config/routes.rb: https://github.com/karakarakaraff/open-to-do/blob/d8bb594e4a2b2fd184b17d8eb3ca21cd18f5febc/config/routes.rb
* db/schema.rb: https://github.com/karakarakaraff/open-to-do/blob/d8bb594e4a2b2fd184b17d8eb3ca21cd18f5febc/db/schema.rb

File: app/models/user.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+  validates :username,
+            length: { minimum: 1, maximum: 100 },
+            presence: true,
+            uniqueness: { case_sensitive: false }
+  validates :password,
+            presence: true,
+            length: { minimum: 6 },
+            if: "password_digest.nil?"
+  validates :password,
```
Comment (bobbilee19) [inline app/models/user.rb]: not sure why repeating some things here and should put all in one

**Comment by bobbilee19** [comment]
all is good except that ^

**Pull request merged**